docs/guide/acquiring_data/hek.rst

```diff
@@ -49,44 +49,24 @@ There is more on the attributes of hek in section 3 of this guide.
 
 So, how many flare detections did the query turn up?
 
-    >>> len(result[0])  # doctest: +REMOTE_DATA
+The result object returned by ``Fido.search`` is a `~.UnifiedResponse` object which contains all the results from any clients used in the search.
+The first thing we need to do is access the results from the HEK client, the only ones for the query we gave.
+
+    >>> len(result['hek'])  # doctest: +REMOTE_DATA
     19
 
-The object returned by the above query is a list of Python dictionary objects.
-Each dictionary consists of key-value pairs that exactly correspond to the parameters listed at http://www.lmsal.com/hek/VOEvent_Spec.html.
+This object is an `astropy.table.Table` object with the columns which correspond to the parameters listed at http://www.lmsal.com/hek/VOEvent_Spec.html.
 
 You can inspect all results very simply:
 
-    >>> result[0] # doctest:+SKIP
-    <sunpy.net.hek.hek.HEKResponse object at ...>
-             SOL_standard          active ... skel_startc2  sum_overlap_scores
-    ------------------------------ ------ ... ------------ --------------------
-    SOL2011-08-08T01:30:04L247C075   true ...         None   12.588887041198964
-    SOL2011-08-08T01:30:04L247C075   true ...         None   12.588887041198964
-    SOL2011-08-08T01:30:04L247C075   true ...         None   12.588887041198964
-    SOL2011-08-09T01:40:04L230C084   true ...         None  37.4574863857972034
-    SOL2011-08-09T02:30:04L319C077   true ...         None  13.6403551847007094
-    SOL2011-08-09T02:30:04L319C077   true ...         None  13.6403551847007094
-    SOL2011-08-09T02:30:04L319C077   true ...         None  13.6403551847007094
-    SOL2011-08-09T07:19:00L296C075   true ...         None                    1
-    SOL2011-08-09T07:19:00L227C090   true ...         None                    0
-    SOL2011-08-09T07:22:38L305C073   true ...         None 0.283032539594587795
-    SOL2011-08-09T07:22:44L305C073   true ...         None                    1
-    SOL2011-08-09T07:48:00L296C073   true ...         None                    0
-    SOL2011-08-09T07:48:00L296C076   true ...         None                    0
-    SOL2011-08-09T07:55:59L305C073   true ...         None                    0
-    SOL2011-08-09T07:59:49L300C077   true ...         None                    0
-    SOL2011-08-09T08:00:03L305C073   true ...         None                    0
-    SOL2011-08-09T08:00:20L305C073   true ...         None                    0
-    SOL2011-08-09T08:00:53L305C073   true ...         None                    0
-    SOL2011-08-09T08:01:21L300C077   true ...         None                    0
+    >>> result['hek']  # doctest: +SKIP
```

Comment:
Was the output too long?

Reply:
It was already skipped, and the output was therefore wrong.